backend/lib/openapi/index.js

```diff
@@ -36,12 +36,13 @@ async function getSchemaDefinitions (user) {
 
   if (_.isEmpty(schemaDefinitions)) {
     // Do not use client of user as the result gets cached and returned to other users
-    const swaggerApi = await dashboardClient.openapi.get()
-    const dereferencedSwaggerApi = await SwaggerParser.dereference(swaggerApi)
+    const openAPIRaw = await dashboardClient.openapi.getGardenerV1Beta1()
+    const dereferencedOpenApi = await SwaggerParser.dereference(openAPIRaw)
 
     const selectedSchemaDefinitions = _
-      .chain(dereferencedSwaggerApi)
-      .get('definitions')
+      .chain(dereferencedOpenApi)
+      .get('components')
+      .get('schemas')
```

Comment:
```suggestion
      .get('components.schemas')
```

Reply:
We should create an issue for adding a time limit to the cache here https://github.com/gardener/dashboard/blob/master/backend/lib/openapi/index.js#L29C7-L29C24

Reply:
https://github.com/gardener/dashboard/issues/1597